.NET/Microsoft.Recognizers.Definitions/English/DateTimeDefinitions.cs

```diff
@@ -193,7 +193,7 @@ public static class DateTimeDefinitions
 		public const string AgoRegex = @"\b(ago|before (?<day>yesterday|today))\b";
 		public const string LaterRegex = @"\b(later|from now|(from|after) (?<day>tomorrow|tmr|today))\b";
 		public const string InConnectorRegex = @"\b(in)\b";
-		public static readonly string WithinNextPrefixRegex = $@"\b(within(\s+the)?(\s+{NextPrefixRegex})?)\b";
+		public static readonly string WithinNextPrefixRegex = $@"\b(within(\s+the)?(\s+(?<next>{NextPrefixRegex}))?)\b";

```

Comment:
This change is used to filter out cases like "within the next 5 days before today"